backend/src/main/java/ai/verta/modeldb/audit_log/AuditLogLocalDAORdbImpl.java

```diff
@@ -31,5 +32,6 @@ public void saveAuditLogs(List<AuditLogLocalEntity> auditLogEntities) {
 
   private void saveAuditLogs(Session session, List<AuditLogLocalEntity> auditLogEntities) {
     auditLogEntities.forEach(session::save);
+    AuditLogInterceptor.increaseAuditCountStatic();
```

Comment:
This allows multiple entities to be added at once and increase just once. It means that a single API can create multiple entries and still pass the interceptor check. That is the incorrect behavior.

Reply:
https://github.com/VertaAI/modeldb/pull/1998/files